modules/common/job/job.go

```diff
@@ -207,6 +208,22 @@ func (j *Job) GetHash() string {
 	return j.hash
 }
 
+// GetTotalAttempts func
+func (j *Job) GetTotalAttempts() int64 {
+	if j.actualJob == nil {
+		return 0
+	}
+	return int64(j.actualJob.Status.Failed)
```

Comment:
should this be `Status.Failed` + `Status.Succeeded` ? like the total attempts of a job would be e.g.  if you have 2 failed and last succeeded so in total you'd have 2 + 1.  Or should rename to `GetTotalFailedAttempts` ?

Reply:
I wanted to register the retries, so renamed to `GetTotalFailedAttempts`